world/shared_task_manager.cpp

```diff
@@ -1300,6 +1302,29 @@ void SharedTaskManager::SendMembersMessageID(
 	}
 }
 
+std::vector<CharacterTaskTimersRepository::CharacterTaskTimers> SharedTaskManager::GetCharacterTimers(
```

Comment:
Why do we vector this when we only return 1?

Reply:
It was just because the previous code used the vector repository result and I didn't want to add more noise modifying code that already relied on it being a vector when I broke it out to a new function